test/ably/rest/restinit_test.py

```
@@ -76,12 +76,12 @@ def test_rest_host_and_environment(self):
         # environment: production
         ably = AblyRest(token='foo', environment="production")
         host = ably.options.get_rest_host()
-        assert "rest.ably.io" == host, "Unexpected host mismatch %s" % host
+        assert "main.realtime.ably.net" == host, "Unexpected host mismatch %s" % host
 
         # environment: other
-        ably = AblyRest(token='foo', environment="sandbox")
+        ably = AblyRest(token='foo', environment="nonprod:sandbox")
         host = ably.options.get_rest_host()
-        assert "sandbox-rest.ably.io" == host, "Unexpected host mismatch %s" % host
+        assert "sandbox.realtime.ably-nonprod.net" == host, "Unexpected host mismatch %s" % host
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `host == "sandbox.realtime.ably-nonprod.net"`

(SIM300)